MIGRATION.md

```diff
@@ -0,0 +1,264 @@
+# Migration Guide
+
+Xrpl.js 3.0 reduces bundle size by 60% and completely removes the need for polyfills for browser applications. It does this with minimal breaking changes. It accomplished this by replacing node specific dependencies with ones which would work in the browser.
+
+The primary breaking change is updating all `Buffer` types to `Uint8Array` since `Buffer` is not supported in the browser. `Buffer` objects can usually be directly replaced with `UInt8Array` as `Buffer` extends `Uint8Array`. The primary difference in functionality is that `Buffer` has additional helper functions. We've listed all functions affected by this change below in the `Uint8Array` section.
+
+This migration guide also applies to:
+`ripple-address-codec` 4.3.1 -> 5.0.0
+`ripple-binary-codec` 1.11.0 -> 2.0.0
+`ripple-keypairs` 1.3.1 -> 2.0.0
+
+# Why update to 3.0?
+
+At a high level:
+
+1. 60% filesize reduction
+2. Simplified setup by removing polyfills
+3. Increased reliability
```

Comment:
Is there a way we could quickly list how reliability is improved? In the previous 2 bullets you said the how, but this bullet is just a statement with no support